src/Atmos/Model/thermo_states.jl

```diff
@@ -2,6 +2,21 @@
 
 export new_thermo_state, recover_thermo_state
 
+"""
+    density(atmos::AtmosModel, state::Vars, aux::Vars)
+
+In the incompressible state, `state.ρ` that is used to
+extract intrinsic values (i.e. `e=state.energy.ρe/state.ρ`)
+is time invariant (by eliminating the tendencies) while
```

Comment:
Why would specific energy be constant in time? And is intrinsic "specific"? -- I do not understand this comment.

Reply:
Yes, I think here @yairchn meant specific when he wrote intrinsic. Also, I think the parenthesis just shows an example of the way we compute a specific variable, but "is time invariant" refers to the state.rho. We will rewrite this for clarity.